packages/core/docs/shopify/configuration.md

```diff
@@ -0,0 +1,69 @@
+# Configuration
```

Comment:
Seems like a link to this file is not added to the sidebar in `packages/core/docs/.vuepress/config.js`

Reply:
Link added :)